pkg/controller/jenkins/jenkins_controller.go

```diff
@@ -12,20 +11,16 @@ import (
 	kappsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	rbacv1 "k8s.io/api/rbac/v1"
+	j "github.com/redhat-developer/openshift-jenkins-operator/pkg/controller/controllerutil"
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/controller"
-	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
-	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
 var (
-	logController      = logf.Log.WithName("jenkins/controller.go")
+	logController      = logf.Log.WithName("jenkins/jenkins_controller.go")
```

Comment:
Can we return the name of the file somehow ?

Reply:
I don't know how yet.
As soon as we find out how, we can do it in a next PR.